tests/cases/fourslash/server/autoImportFileExcludePatterns1.ts

```diff
@@ -2,19 +2,19 @@
 
 // @module: commonjs
 
-// @Filename: /project/node_modules/aws-sdk/package.json
+// @Filename: /home/src/workspaces/project/node_modules/aws-sdk/package.json
```

Comment:
I can understand if we don’t want to test files in the disk root like `/tsconfig.json`, but in this test, everything was inside `/project/*`. Did this test actually need to change, or was this just a bulk find and replace? Is the `/home/src/workspaces` prefix significant?

Reply:
This does need to change because we don’t watch directories with /soemthing/anotherdir/

Reply:
Just as an example if you did not start file name with "/" they are relative to "/tests/case/fourslash" and i didnt need to change those

Reply:
Directories have to be 2 or more levels deep?

Reply:
Yes. https://github.com/microsoft/TypeScript/blob/main/src/compiler/resolutionCache.ts#L307